internal/compiler/lib.rs

```diff
@@ -92,7 +92,12 @@ pub struct CompilerConfiguration {
     /// The callback should open the file specified by the given file name and
     /// return an future that provides the text content of the file as output.
     pub open_import_fallback: Option<
-        Rc<dyn Fn(String) -> Pin<Box<dyn Future<Output = Option<std::io::Result<String>>>>>>,
+        Rc<
+            dyn Fn(
+                String,
+            )
+                -> Pin<Box<dyn Future<Output = Option<std::io::Result<(String, Option<i32>)>>>>>,
```

Comment:
We have an alias for that, let's use it.

```suggestion
                -> Pin<Box<dyn Future<Output = Option<std::io::Result<(String, SourceFileVersion)>>>>>,
```